test/features/operator/install_explainability.feature

```diff
@@ -6,17 +6,55 @@ Feature: Kogito Explainability service
     Given Namespace is created
 
   @smoke
-  # Disabled as long as https://issues.redhat.com/browse/KOGITO-3176 is not solved
-  @disabled
```

Comment:
I've enabled this test adding a workaround (manually creating the kafka topic in a step). Let me know your thoughts about it.

Reply:
Hi @Sgitario , my 50 cents is that this should not be intented as workaround. imo this is the proper solution to the problem: the user should provide the kafka instance + create the kafka topic resources, and this is exactly what you have implemented here.  
If the user does not want to create manually the topics, he has to enable the auto-creation of topics in the kafka broker and all the services should be deployed.

Reply:
For the tests, for now, Kogito Operator creates the Kafka CR with auto-creation topics. If the BDD uses the same approach is not a problem IMO

Reply:
Ok, I've removed the workaround comment and leave it like this. My only concern now is if the ticket KOGITO-3176 needs fixing or not :/

Reply:
Hi @Sgitario @ricardozanini , I don't think we are going to implement on the trusty service a logic to manually create the topic by the consumer. If I'm not wrong, such feature has been removed by the "official" smallrye library we are using and I don't think we should re-introduce it on our side. 
We have two options: 
1) remove the test on the standalone trusty service
2) create the topics like you proposed (I would go for this)

Reply:
To provide further context, the ticket is https://issues.redhat.com/browse/KOGITO-3176 and is causing that:
- When deploying the trusty service, but not the explainability service - the trusty service is not starting because the kafka topic "trusty-explainability-request" does not exist (this topic is automatically created when deploying the explainability service. 
- When deploying the explainability service, but not the trusty service - the explainability service is not starting because the kafka topic "trusty-explainability-result" does not exist (this topic is created when deploying the trusty service).
- When deploying the explainability and trusty services both, everything works fine. 

I think it might be an architecture issue here as trusty and explainability services depend on each other. Some other services like Jobs Service creates the kafka topic: https://github.com/kiegroup/kogito-apps/blob/05cca086194fa14d80a725fb381feb7b586b3496/jobs-service/src/main/java/org/kie/kogito/jobs/service/stream/KafkaConfiguration.java#L86. I'm not saying that this is the right approach, but it needs further discussion and broader agreement. 

@r00ta I'm moving this discussion thread to the JIRA ticket.